### PR TITLE
Add the VP9 to the game

### PR DESCRIPTION
**Pull request by BalthazarArgall**
#### Summary
Content "Add the VP9 gun"

#### Purpose of change
I was asking for some gun to implement a few days ago and Kevin himself mentioned the VP9. I checked Gunbroker and indeed the gun is very popular yet doesn't exist in the game.

#### Describe the solution
Add it.

#### Testing
Not yet, I need to fix some issues with my VS first.

#### Additional context
Gunbroker hits for reference:
Untitled

### CODE REVIEW

File: data/json/items/gun/9mm.json [hunk 1]
```diff
@@ -1332,5 +1332,54 @@
...
     ],
     "melee_damage": { "bash": 9 }
+  },
+  {
+    "id": "vp9",
+    "copy-from": "pistol_base",
+    "type": "GUN",
+    "weapon_category": [ "AUTOMATIC_PISTOLS" ],
+    "name": { "str_sp": "9mm semi automatic pistol" },
+    "description": "A semi automatic pistol, chambered in 9mm .",
```
Comment (TheShadowFerret) [inline data/json/items/gun/9mm.json:1342]:
```suggestion
    "description": "A semi-automatic pistol, chambered in 9mm.",
```

File: data/json/items/magazine/9mm.json [hunk 2]
```diff
@@ -1566,5 +1566,62 @@
     "reload_time": 160,
     "flags": [ "MAG_BULKY" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "9mm": 50 } } ]
+  },
+  {
+    "id": "vp9mag_9x19mm_10rd",
+    "type": "MAGAZINE",
+    "looks_like": "glock17_17",
+    "name": { "str": "VP9 10-round magazine" },
+    "description": "A 10 rounds magazine for use with the VP9 and the P30.",
```
Comment (TheShadowFerret) [inline data/json/items/magazine/9mm.json:1575]:
```suggestion
    "description": "A 10-magazine for use with the VP9 and the P30.",
```

File: data/json/items/magazine/9mm.json [hunk 3]
```diff
@@ -1566,5 +1566,62 @@
...
+    "ammo_type": [ "9mm" ],
+    "flags": [ "MAG_COMPACT" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "9mm": 10 } } ]
+  },
+  {
+    "id": "vp9mag_9x19mm_15rd",
+    "copy-from": "vp9mag_9x19mm_10rd",
+    "type": "MAGAZINE",
+    "name": { "str": "VP9 15-round magazine" },
+    "description": "A 15 rounds magazine for use with the VP9 and the P30.",
```
Comment (TheShadowFerret) [inline data/json/items/magazine/9mm.json:1593]:
```suggestion
    "description": "A 15-round magazine for use with the VP9 and the P30.",
```

File: data/json/items/magazine/9mm.json [hunk 4]
```diff
@@ -1566,5 +1566,62 @@
...
+    "price": "50 USD",
+    "price_postapoc": "3 USD",
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "9mm": 15 } } ]
+  },
+  {
+    "id": "vp9mag_9x19mm_17rd",
+    "copy-from": "vp9mag_9x19mm_10rd",
+    "type": "MAGAZINE",
+    "name": { "str": "VP9 17-round magazine" },
+    "description": "A 17 rounds magazine for use with the VP9 and the P30.",
```
Comment (TheShadowFerret) [inline data/json/items/magazine/9mm.json:1606]:
```suggestion
    "description": "A 17-round magazine for use with the VP9 and the P30.",
```

File: data/json/items/magazine/9mm.json [hunk 5]
```diff
@@ -1566,5 +1566,62 @@
...
+    "price": "50 USD",
+    "price_postapoc": "3 USD",
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "9mm": 17 } } ]
+  },
+  {
+    "id": "vp9mag_9x19mm_20rd",
+    "copy-from": "vp9mag_9x19mm_10rd",
+    "type": "MAGAZINE",
+    "name": { "str": "VP9 20-round magazine" },
+    "description": "A 20 rounds magazine for use with the VP9 and the P30.",
```
Comment (TheShadowFerret) [inline data/json/items/magazine/9mm.json:1619]:
```suggestion
    "description": "A 20-round magazine for use with the VP9 and the P30.",
```

**Review by TheShadowFerret — COMMENTED** [review]
>Mentions the P30 as well
>No P30 added

😿

**Comment by Night-Pryanik** [comment]
Closing as stale. If you wish to continue working on this, ping me to reopen.

**Pull request closed without merging**